Runtimes/Supplemental/Distributed/CMakeLists.txt

```diff
@@ -57,47 +72,53 @@ option(${PROJECT_NAME}_ENABLE_LIBRARY_EVOLUTION "Generate ABI resilient runtime
 option(${PROJECT_NAME}_ENABLE_PRESPECIALIZATION "Enable generic metadata prespecialization"
   ${SwiftCore_ENABLE_PRESPECIALIZATION})
 
-add_library(swiftDistributed
-  DistributedActor.cpp
-  DistributedActor.swift
-  DistributedActorSystem.swift
-  DistributedAssertions.swift
-  DistributedDefaultExecutor.swift
-  DistributedMacros.swift
-  DistributedMetadata.swift
-  LocalTestingDistributedActorSystem.swift)
-
-target_compile_options(swiftDistributed PRIVATE
+add_compile_options(
   $<$<COMPILE_LANGUAGE:Swift>:-explicit-module-build>
   $<$<COMPILE_LANGUAGE:Swift>:-nostdlibimport>
-  $<$<COMPILE_LANGUAGE:Swift>:-parse-stdlib>
-  $<$<COMPILE_LANGUAGE:Swift>:SHELL:-enable-experimental-feature AllowUnsafeAttribute>
   $<$<COMPILE_LANGUAGE:Swift>:-strict-memory-safety>
-  "$<$<COMPILE_LANGUAGE:Swift>:SHELL:-library-level api>"
```

Comment:
This one was dropped, but I think it gets added by the `emit_swift_interface` call below.